### ISSUE TITLE
How can i use AdaBoostClassifier using 'base_estimator' parameter?

### ISSUE THREAD

**Issue by pvachill** [EpistasisLab/tpot#750]
I tried to use AdaBoostClassifer and i succeded in the scenario that i dont use the 'base_estimator' parameter. But when i try to set 'base_estimator' it fails. I tried two ways:
1)
 'sklearn.ensemble.AdaBoostClassifier': {
        'base_estimator': {
            'sklearn.naive_bayes.GaussianNB': {
            }
        }
 },

Which leads to: **RuntimeError: A pipeline has not yet been optimized. Please call fit() first.**

2 and a half)
'sklearn.ensemble.AdaBoostClassifier': {
        'base_estimator': ['sklearn.naive_bayes.GaussianNB']  or ['sklearn.naive_bayes.GaussianNB()']
}

Which both lead to: **RuntimeError: There was an error in the TPOT optimization process. This could be because the data was not formatted properly, or because data for a regression problem was provided to the TPOTClassifier object. Please make sure you passed the data to TPOT correctly.**

But if use AdaBoostClassifier with any parameter except base_estimator it runs properly.
 'sklearn.ensemble.AdaBoostClassifier': {
     'param1': [something],
     'param2': [something else]
}  

Thanks you,
Achilleas.

**Comment by weixuanfu** [comment]
Thank you for reporting this issue here. I just posted a PR #751 to fix this issue and we will release a new version of TPOT soon with this fix. For now, there are two work-arounds:

1. install the PR with fix via the command below. But it is noted that it is based on development branch.
```
pip install --upgrade --no-deps --force-reinstall git+https://github.com/weixuanfu/tpot.git@issue750
```

2. add a useless parameter (`'priors': [None]` in the demo below) to bypass the bug.

```python
# coding: utf-8
from tpot import TPOTClassifier
from sklearn.datasets import load_digits
from sklearn.model_selection import train_test_split

digits = load_digits()
X_train, X_test, y_train, y_test = train_test_split(digits.data, digits.target,
                                                    train_size=0.75, test_size=0.25)

tpot_config = {
                                                         
'sklearn.ensemble.AdaBoostClassifier': {
'base_estimator': {
'sklearn.naive_bayes.GaussianNB': {
'priors': [None]
}
}
}
}

tpot = TPOTClassifier(generations=5, population_size=20, verbosity=2,
                      config_dict=tpot_config)
tpot.fit(X_train, y_train)
print(tpot.score(X_test, y_test))
```

**Comment by weixuanfu** [comment]
This issue should be fixed in TPOT 0.9.4. Please feel free to reopen this issue if you have any questions.
